Makefile

```diff
@@ -9,7 +9,8 @@ lint:
 
 vet:
 	@echo "⚠️ Warning: the following only works with go >= 1.14" && \
-	go install ./internal/mcheck && \
+	go get go.dedis.ch/dela/internal/mcheck && \
+	go install go.dedis.ch/dela/internal/mcheck && \
```

Comment:
I'm also not sure I understand this change.

Reply:
thanks